releasenotes/notes/add_aer_state-f8bcee6d844f6dad.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add new class AerState to support access simulator state from python directly.
+    This class is just for internal usages and end users can access it through
+    other classes such as AerStatevector.
```

Comment:
If it's private we don't need a release note since release notes are just for documenting changes for users in the release.